### DISCUSSION TITLE
A mop

### DISCUSSION THREAD

**Discussion by Mamuts123** [FakeFishGames/Barotrauma#12315] [Ideas and suggestions]
Hold right click and press E to clean walls and floors, like the spray gun with ethanol. Use left click to attack, deals 2-5 Blunt force trauma. 25 Mk at outposts. Used by assistant. Is a mop.

**Comment by Hazmat616** [comment]
You've only scratched the surface with this idea. Makes a smap sound when wet and can possibly be used in some clown related shenanigans.
